app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,116 @@
 <!DOCTYPE html>
```

Comment:
Some things I would recommend here:
- Make use of partials with good naming conventions - like navigation, head or something along those lines
- Use a better templating language - slim or haml are great. Whatever you are more comfortable with.